src/styles/components/modal/styles.less

```diff
@@ -1,6 +1,22 @@
 & when (@modal-enabled) {
 
+  // using flexbox for centering fixes the blurry subpixel 
+  // rendering of dialog modals on low resolution screens
+  .modal-wrapper {
+    align-items: center;
+    bottom: 0;
+    display: flex;
+    justify-content: center;
+    left: 0;
+    right: 0;
+    top: 0;
+  }
+
   .modal {
```

Comment:
@mperrotti , the failing integration test is fixed if we change this to `modal-small` and I think this won't affect the actual big that we are fixing, since it only happens for small modals anyway.

Reply:
since my screencast-app fails after an update earlier this day i'll try to describe what's happening:

the arrow-right on an image gallery within the packagesDetail is failing. you can have a look at that here: https://soak114s.testing.mesosphe.re/#/catalog/packages/instana-agent?version=1.0.5.

when looking for packages that have screenshots i could reproduce the error from CI on soak. so it looks like we may "just" have made things worse.